README.md

```diff
@@ -137,19 +137,22 @@ Other Style Guides
       count += 1;
     }
     ```
-
   <a name="references--block-scope"></a><a name="2.3"></a>
-  - [2.3](#references--block-scope) Note that both `let` and `const` are block-scoped.
+  - [2.3](#references--block-scope) Note that both `let` and `const` are block-scoped. Whereas `var` is function scoped.
 
     ```javascript
     // const and let only exist in the blocks they are defined in.
     {
       let a = 1;
       const b = 1;
+      var c = 1;
     }
     console.log(a); // ReferenceError
     console.log(b); // ReferenceError
+    console.log(c); // Prints 1
     ```
+    In above code, you can see that the `a` and `b` will generate reference error and `c` prints the number. It is because `a` and `b` are block scoped while `c` is not.
```

Comment:
```suggestion
    In above code, you can see that the `a` and `b` will generate a ReferenceError, and `c` prints the number. This is because `a` and `b` are block scoped, while `c` is scoped to the containing function.
```